app/models/journeyDomain/transportMeans/TransportMeansDomain.scala

```diff
@@ -22,68 +22,123 @@ import controllers.transportMeans.routes
 import models.SecurityDetailsType.NoSecurityDetails
 import models.domain._
 import models.journeyDomain.{JourneyDomainModel, Stage}
-import models.transportMeans.{BorderModeOfTransport, InlandMode}
+import models.transportMeans.BorderModeOfTransport
+import models.transportMeans.BorderModeOfTransport.ChannelTunnel
 import models.{Mode, Phase, UserAnswers}
-import pages.external.SecurityDetailsTypePage
+import pages.external.{OfficeOfDepartureInCL010Page, SecurityDetailsTypePage}
 import pages.preRequisites.ContainerIndicatorPage
-import pages.transportMeans._
+import pages.transportMeans.{
+  AddActiveBorderTransportMeansYesNoPage,
+  AddBorderModeOfTransportYesNoPage,
+  AddDepartureTransportMeansYesNoPage,
+  BorderModeOfTransportPage
+}
 import play.api.mvc.Call
 
-case class TransportMeansDomain(
-  transportMeansDeparture: Option[TransportMeansDepartureDomain],
-  borderModeOfTransport: Option[BorderModeOfTransport],
-  transportMeansActiveList: Option[TransportMeansActiveListDomain]
-) extends JourneyDomainModel {
+sealed trait TransportMeansDomain extends JourneyDomainModel {
 
   override def routeIfCompleted(userAnswers: UserAnswers, mode: Mode, stage: Stage, phase: Phase): Option[Call] =
     Option(routes.TransportMeansCheckYourAnswersController.onPageLoad(userAnswers.lrn, mode))
+
 }
 
 object TransportMeansDomain {
 
   implicit def userAnswersReader(implicit phaseConfig: PhaseConfig): UserAnswersReader[TransportMeansDomain] =
-    InlandModePage.reader.flatMap {
-      case InlandMode.Mail =>
-        UserAnswersReader(TransportMeansDomain(None, None, None))
-      case _ =>
-        (
-          transportMeansDepartureReader,
-          borderModeOfTransportReader,
-          transportMeansActiveReader
-        ).tupled.map((TransportMeansDomain.apply _).tupled)
-    }
```

Comment:
This logic looks to have been wiped out? Has it been moved elsewhere? Or is it no longer needed?

Reply:
No longer required. Checked with Matt.  If inland mode method is selected as 'mail' earlier in the journey then these sections are bypassed, It isn't even a radio option on mode crossing border

Reply:
@kenneth-rayner but then if we're not reading that page anywhere it won't get asked?

Reply:
@jackheslop96 we have rectified this